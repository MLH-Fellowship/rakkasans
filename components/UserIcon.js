import React from "react";
import { View, StyleSheet } from "react-native";

import Colors from "../constants/Colors";
import Icon from "./Icon";

export default function UserIcon() {
  return (
    <View style={styles.icon}>
      <Icon name={"plus"} color={Colors.darkGray} size={50} />
    </View>
  );
}

const styles = StyleSheet.create({
  icon: {
    margin: 20,
    marginBottom: 35,
    width: 100,
    height: 100,
    borderRadius: 50,
    borderWidth: 1,
    borderColor: Colors.darkGray,
    backgroundColor: Colors.lightGray,
    justifyContent: "center",
    alignItems: "center",
  },
});
