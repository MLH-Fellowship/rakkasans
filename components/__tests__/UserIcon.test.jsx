// jest.useFakeTimers();
import React from 'react';
import renderer from 'react-test-renderer';

import UserIcon from '../UserIcon';

jest.useFakeTimers();

it('renders correctly', () => {
  const tree = renderer.create(<UserIcon />).toJSON();
  expect(tree).toMatchSnapshot();
});
