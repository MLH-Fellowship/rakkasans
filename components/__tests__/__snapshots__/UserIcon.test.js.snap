// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#efefef",
      "borderColor": "#333333",
      "borderRadius": 50,
      "borderWidth": 1,
      "height": 100,
      "justifyContent": "center",
      "margin": 20,
      "marginBottom": 35,
      "width": 100,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "width": 50,
      }
    }
  >
    <Text />
  </View>
</View>
`;
